### DISCUSSION TITLE
What features would you like to have on WorkLenz ?

### DISCUSSION THREAD

**Discussion by kalinga777** [Worklenz/worklenz#18] [Ideas]
Dear contributors and users,

We would like to know what features would you like to have on WorkLenz ? Feel free to comment :)

**Comment by ilovepancakes95** [comment]
An integration/extension for Mattermost would be awesome. https://mattermost.com/

Their current "Boards" extension is rather simple/lacking in many features Worklenz seems to have.

**Reply by kalinga777** [reply]
Thanks will try to add

**Comment by Riprock** [comment]
Integration with other auth methods, LDAP/AD support mission is the only thing that is holding me back from staring to adopt this in my team

**Reply by selmamehdi48** [reply]
An essential feature, especially for our enterprise that utilizes Active Directory (AD) for user management.

**Reply by selmamehdi48** [reply]
@kalinga777

**Comment by selmamehdi48** [comment]
Language selector: French, Arabic...etc

**Reply by kalinga777** [reply]
This is coming in our new version

**Comment by ilovefreesw** [comment]
Cover images for cards in kanban board. The only thing that kept me away from using it. Or maybe the custom SMTP. AWS keep rejecting to move me away from sandbox mode for SES.

**Comment by lewebsimple** [comment]
Webhooks and / or an API to connect with external platforms like Quickbooks online, etc

**Comment by devoluxbv** [comment]
More advanced time\money tracking. 

Add budget per user per month, working hours per month, add price per hour, price per hour for overwork, add sell price per hour. 

See report based on this data.

**Reply by kalinga777** [reply]
@devoluxbv we are building this feature as of now

**Comment by devoluxbv** [comment]
Ability to export some kind of backup like projects, tasks, comments. 
It is scary to use system without control of backup. 

I am SaaS user

**Reply by kalinga777** [reply]
Thanks . Will add this feature into our pipeline

**Comment by YushengLi** [comment]
Ability to manage expense on Projects alongside timesheet

**Reply by kalinga777** [reply]
@YushengLi currently we are building the feature and it will be a premium plugin for self hosted users

**Comment by 002gRTEah3mRYH** [comment]
Hello,
A desktop app
Thanks
